core/web/resolver/spec.go

```diff
@@ -472,8 +473,8 @@ func (r *OCR2SpecResolver) BlockchainTimeout() *string {
 }
 
 // ContractAddress resolves the spec's contract address.
-func (r *OCR2SpecResolver) ContractAddress() string {
-	return r.spec.ContractAddress.String()
+func (r *OCR2SpecResolver) ContractID() string {
+	return r.spec.ContractID.String
```

Comment:
Why is it a nullable type when its not-null in the database?

Reply:
Are you refering to the spec field definition here because the GQL schema defines it as required

Reply:
No, I mean it looks as though `ContractID` is a `null.String`. But it doesn't need to be, it's `NOT NULL` in the database so we can just use a plain ol' `string` type.

We should never be reading `null.String``String` fields without checking validity anyway.

Reply:
agreed, it should just be a string